js/core/polyfills/matches.js

```diff
@@ -1,23 +1,27 @@
 "use strict";
 
-var Element = require("../dom_adapter").getWindow().Element;
+var window = require("../dom_adapter").getWindow();
+var matches,
+    getMatches = function() {
+        return window.Element.prototype.matches ||
```

Comment:
Try to wrap it into the [callOnce](https://github.com/DevExpress/DevExtreme/pull/2504) method